pyproject.toml

```diff
@@ -111,6 +111,11 @@ module = [
 ]
 ignore_errors = true
 
+[[tool.mypy.overrides]]
+module = ['tests.*']
+disallow_untyped_decorators = false
```

Comment:
this is only disabled due to `pytest` not currently being in `mypy`'s environment

needs https://github.com/python-poetry/poetry/pull/5279